### PR TITLE
Update cmaps

### PR DESCRIPTION
**Pull request by dougfelt**

### CODE REVIEW

**Comment by dougfelt** [comment]
fixes #433

File: nototools/data/noto_cmap_phase3.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <tr>Bamu,Bamum,661,0000 000d 0020 00a0 a6a0-a6f7 16800-16a38,0,</tr>
     <tr>Bass,Bassa Vah,45,0000 000d 0020 0022 002c 002e 00a0 201c-201d 16ad0-16aed 16af0-16af5,-1,</tr>
     <tr>Batk,Batak,74,0000 000d 0020 00a0 00d7 1bc0-1bf3 1bfc-1bff 200b-200d 2012-2015 2022 25cc 25fb-25fe,-1,</tr>
-    <tr>Beng,Bengali,157,0000 000d 0020-0023 0025 0027-003f 005b-005f 007b-007e 00a0 00ad 00d7 00f7 0951-0952 0964-0965 0980-0983 0985-098c 098f-0990 0993-09a8 09aa-09b0 09b2 09b6-09b9 09bc-09c4 09c7-09c8 09cb-09ce 09d7 09dc-09dd 09df-09e3 09e6-09fb 200b-200d 2010 2013-2014 2018-2019 201c-201d 2026 20b9 2212 25cc a8f1,-1,</tr>
+    <tr>Beng,Bengali,157,0000 000d 0020-0023 0025 0027-003f 005b-005f 007b-007e 00a0 00ad 00d7 00f7 02bc 0951-0952 0964-0965 0980-0983 0985-098c 098f-0990 0993-09a8 09aa-09b0 09b2 09b6-09b9 09bc-09c4 09c7-09c8 09cb-09ce 09d7 09dc-09dd 09df-09e3 09e6-09fb 200b-200d 2010 2013-2014 2018-2019 201c-201d 2026 20b9 2212 25cc,-1,</tr>
```
Comment (jungshik) [inline nototools/data/noto_cmap_phase3.xml]: Drop U+A8F1?  I thought we had decided to keep it in Bengali?  Oh. I saw the comment below in the code. Ok.

Comment (jungshik) [inline nototools/data/noto_cmap_phase3.xml]: I guess U+02BC (modifier apostrophe) is necessary somehow.

File: nototools/noto_cmap_reqs.py [hunk 2]
```diff
@@ -355,6 +355,17 @@ def _primary_script_for_block(block):
...
 
+def _remove_unicode_assignments(cmap_ops):
+  """The starting point is based on the script and script extensions data from
+  Unicode.  Sometimes the assignments seem premature."""
+  cmap_ops.phase('remove unicode assignments')
+
+  # Jelle says A8F1 makes no sense for Bengali since other characters needed
+  # for cantillation are not defined.  Unicode script extensions assign it to
+  # Deva and Beng, leave it for Deva.
+  cmap_ops.remove(0xa8f1, 'Beng')
```
Comment (jungshik) [inline nototools/noto_cmap_reqs.py]: Perhaps, time to file a feedback against the Unicode.

File: nototools/noto_cmap_reqs.py [hunk 3]
```diff
@@ -2042,6 +2065,21 @@ def _generate_script_extra(script_to_chars):
...
+   """,
+   # Data
+   """
+   # Combining Diacritical Marks
+   0301  # COMBINING ACUTE ACCENT
+   0304  # COMBINING MACRON
+   030B  # COMBINING DOUBLE ACUTE ACCENT
+   0358  # COMBINING DOT ABOVE RIGHT
+   # Geometric Shapes
+   25CC  # DOTTED CIRCLE
```
Comment (jungshik) [inline nototools/noto_cmap_reqs.py]: I wonder if there's any other script that needs U+25CC but does not currently have. Basically, any 'complex' script would need it, wouldn't it?

Comment (dougfelt) [inline nototools/noto_cmap_reqs.py]: Yeah, I'm going to add that but decided to keep it as a separate cl.  These changes originate with Monotype, any 25CC changes would originate with us.

File: nototools/noto_cmap_reqs.py (hunk 2 above)
Comment (dougfelt) [inline nototools/noto_cmap_reqs.py]: There seems to be stuff going on in this space.  There's a proposal from Jan of this year to add "combining bengali sign avagraha" to Bengali with "the same properties as A8F1".  I asked Roozbeh if he knew more about it.  We'll see I guess.
